tests/integration-tests/tests/ad_integration/test_ad_integration.py

```diff
@@ -604,7 +710,7 @@ def test_ad_integration(
     scheduler_commands = scheduler_commands_factory(remote_command_executor)
     assert_that(NUM_USERS_TO_TEST).is_less_than_or_equal_to(NUM_USERS_TO_CREATE)
     users = []
-    for user_num in range(1, NUM_USERS_TO_TEST + 1):
+    for user_num in range(1, NUM_USERS_TO_TEST):
```

Comment:
Why do the range start from 1?

Reply:
Good catch! It is a change I've introduced during debugging, will remove it. Thanks a lot!

Reply:
I think it was like that just to have user names starting with 1 rather than 0.
I will check if it needed for some other reasons. Otherwise, I will make it start from 0.

Reply:
Seems there is no need to start the numeration that way. Will change it in the next rev